recipes/FunctionalPlus/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
If you are building this tool for Windows, you will need to have build script for Windows, named `bld.bat` as well, https://docs.conda.io/projects/conda-build/en/latest/resources/build-scripts.html.

Reply:
Thanks again. Just added it.